distribution/integration/tests-integration/user-core/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
+log4j.rootLogger=ERROR, stdout
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
```

Comment:
Shall we also log to a target/logs/ folder?

Reply:
Addressed